### ISSUE TITLE
Issue with "sh models/download_models.sh" command

### ISSUE THREAD

**Issue by Mockapapella** [jcjohnson/neural-style#160]
OK so I installed all the dependencies correctly (as far as I can tell, at least), and I installed CUNN. I type:

`sudo sh models/download_models.sh`

What outputs in this:

`sh: 0: Can't open models/download_models.sh`

I have no idea what's going on here. Can someone please explain what I'm doing wrong?

**Comment by hackwave** [comment]
Yes. The only issue you'll run into is that for one of the installs it'll need to use gcc 4.9 to compile, it'll tell you in the error. Ubuntu 15 is on gcc 5. so when you run into that error change your .bashrc profile to include "export cc=/usr/bin/gcc-4.9" and "export "CXX=/usr/bin/g++-4.9".  Refresh bashrc. then run your install.   After it's all done switch it back to gcc 5 before you start anymore installs.

Didn't have any other issues.
